prometheus_pgbouncer_exporter/collector.py

```diff
@@ -7,7 +7,7 @@
 
 
 class PgbouncersMetricsCollector():
-    def __init__(self, configs: List[PgbouncerConfig]):
+    def __init__(self, configs: List[PgbouncerConfig]):        
```

Comment:
Extra spaces at the end of the line

```suggestion
    def __init__(self, configs: List[PgbouncerConfig]):
```